admin/tools/docker/4_build-configured-images.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+# LSST Data Management System
+# Copyright 2014 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+# Create Docker images containing Qserv master and worker instances
+
+# @author  Fabrice Jammes, IN2P3/SLAC
+
+set -e
+set -x
+
+DOCKER_IMAGE="qserv/qserv:dev"
+
+usage() {
+  cat << EOD
+
+Usage: `basename $0` [options] host
+
+  Available options:
+    -h          this message
+    -i          Docker image to be used as input, default to $DOCKER_IMAGE
```

Comment:
if `-i option ` takes a parameter you can add to help string for clarity:

        -i image    Docker image to be used as input, default to $DOCKER_IMAGE